content/en/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function.md

```diff
@@ -21,7 +21,7 @@ To start collecting logs from your AWS services:
 2. [Enable logging](#enable-logging-for-your-aws-service) for your AWS service (most AWS services can log to a S3 bucket or CloudWatch Log Group).
 3. [Set up the triggers](#set-up-triggers) that cause the Forwarder Lambda to execute when there are new logs to be forwarded. There are two ways to configure the triggers.
 
-**Note**: If you are in AWS us-east-1 region, leverage [Datadog-AWS Private Link][2].
+**Note**: If you are in AWS `us-east-1` region, leverage [Datadog-AWS Private Link][2].
 
 **Note**: Cloudformation creates an IAM policy which includes KMS:Decrypt for all resources. It is known that this does not align with Security hub best practice. The reason this permission is used is to decrypt objects from KMS encrypted S3 buckets to set up Lambda function, and which KMS key is used to encrypt the S3 buckets cannot be predicted. You can safely delete this permission after the installation successfully finished.
```

Comment:
```suggestion
If you are in the AWS `us-east-1` region, leverage [Datadog-AWS Private Link][2].

CloudFormation creates an IAM policy which includes KMS:Decrypt for all resources, and does not align with AWS Security Hub's best practice. This permission is used to decrypt objects from KMS-encrypted S3 buckets to set up Lambda function, and which KMS key is used to encrypt the S3 buckets cannot be predicted. You can safely delete this permission after the installation successfully finished.
```

Reply:
Thanks for these suggestions, I hadn't looked into this paragraph as it wasn't part of the update. I've added most of your suggestions but left the **Note:**, as this may be intentional to bring attention to this. Could you clarify the reason for removing the Note, so that I can follow up to confirm with the AWS team?

Reply:
I think these changes are fine but we should leave the **Note:** unless there is a strong reason to remove